website/blog/2021-09-29-release-apache-apisix-2.10-en.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Release Apache APISIX 2.10.0"
+author: "Zexuan Luo"
+authorURL: "https://github.com/spacewander"
+authorImageURL: "https://avatars.githubusercontent.com/u/4161644?v=4"
+keywords:
+- Apache APISIX
+- API Gateway
+- Release Notes
+- APISIX 2.10
+description: Apache APISIX 2.10 is released!
```

Comment:
```suggestion
description: Apache APISIX 2.10.0 is released!
```

how about add the related `the first LTS version` to the description?

Reply:
fixed, replace all 2.10.0 with 2.10